internal/cmap/cmap.go

```diff
@@ -21,6 +21,9 @@ const (
 
 	// MissingCodeRune replaces runes that can't be decoded. '\ufffd' = �. Was '?'.
 	MissingCodeRune = '\ufffd' // �
+
+	// MissingCodeRune replaces strings that can't be decoded.
```

Comment:
Should be MissingCodeString
Maybe it would be enough to have
// MissingCodeString is string(MissingCodeRune).

Reply:
Done